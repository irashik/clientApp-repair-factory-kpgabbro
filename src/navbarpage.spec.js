import React from 'react';
//import { shallow, render } from 'enzyme';
import renderer from 'react-test-renderer';


import NavbarPage  from './navbarpage';

import Container from 'react-bootstrap/Container';
import Row from 'react-bootstrap/Row';
import { BrowserRouter as Router, 
          Route, 
          Switch,
       } from 'react-router-dom';


    //    import { LinkContainer } from  'react-router-bootstrap';
    //    import Button from 'react-bootstrap/Button';
    //    import ButtonGroup from 'react-bootstrap/ButtonGroup';
    //    import Nav from 'react-bootstrap/Nav';
    //    import Navbar from 'react-bootstrap/Navbar';


       jest.mock('react-router-bootstrap');
       jest.mock('react-bootstrap/Button');
       jest.mock('react-bootstrap/ButtonGroup');
      // jest.mock('react-bootstrap/Nav');
       // jest.mock('react-bootstrap/Navbar');



describe("<NavbarPage />", () => {
    test('should render correctly', () => {

        const component = shallow(<NavbarPage />)
        expect(component).toMatchSnapshot();



        
    });
});
