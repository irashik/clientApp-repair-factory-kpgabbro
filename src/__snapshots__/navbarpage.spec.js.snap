// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarPage /> should render correctly 1`] = `
<Navbar
  className="navbar-custom"
  collapseOnSelect={false}
  expand="sm"
  variant="light"
>
  <NavbarBrand
    href="/"
  >
    Журнал ремонтов
  </NavbarBrand>
  <NavbarToggle
    aria-controls="basic-navbar-nav"
    label="Toggle navigation"
  />
  <NavbarCollapse
    id="basic-navbar-nav"
  >
    <Nav
      className="mr-auto"
      fill={false}
      justify={false}
    >
      <NavLink
        disabled={false}
        href="/inputData"
        id="inputData"
      >
        Ввод данных
      </NavLink>
      <NavLink
        disabled={false}
        href="/reportEquipment"
        id="reportEquipment"
      >
        Отчет по оборудованию
      </NavLink>
      <NavLink
        disabled={false}
        href="/repairPlan"
        id="repairPlan"
      >
        План ремонтов
      </NavLink>
      <NavLink
        disabled={false}
        href="/bidRequest"
        id="bidRequest"
      >
        Заявки снабжение
      </NavLink>
      <AuthButton />
    </Nav>
  </NavbarCollapse>
</Navbar>
`;
